BedrockCommand.cpp

```diff
@@ -240,12 +253,14 @@ void BedrockCommand::finalizeTimingInfo() {
     }
 
     SINFO("command '" << methodName << "' timing info (ms): "
-          "prePeek: " << prePeekTotal/1000 << " (count: " << prePeekCount << "), "
+          "prePeek:" << prePeekTotal/1000 << " (count: " << prePeekCount << "), "
           "peek:" << peekTotal/1000 << " (count:" << peekCount << "), "
           "process:" << processTotal/1000 << " (count:" << processCount << "), "
           "postProcess:" << postProcessTotal/1000 << " (count:" << postProcessCount << "), "
           "total:" << totalTime/1000 << ", "
-          "unaccounted:" << unaccountedTime/1000 <<
+          "unaccounted:" << unaccountedTime/1000 << ", "
+          "blockingCommitThreadTime:" << blockingCommitThreadTime/1000 << ", "
+          "exclusiveTransactionLockTime:" << exclusiveTransactionLockTime/1000 <<
```

Comment:
@flodnv  Is this fine or does it feel like it just will get in the way when reading the timing info?

Reply:
I think this is fine